apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -108,7 +108,11 @@
             text="Controls Viewed"
             sort="disabled"
             :viewed-header="true"
-            :number-of-viewed-controls="viewedControlIds.length"
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/138dc7b0-f94e-419f-9252-432738260135)

seems to have broken the counting entirely within the deploy preview where things aren't being hidden after being marked as viewed nor as is the count being adjusted

Reply:
It was a reactivity issue with Sets, I moved backed to array and implemented a fix.  It should be working now

Reply:
It's working